### PR TITLE
Update i-find, i-restock and DG storage class image and help window

### LINKED ISSUE #177: [PE-D] Not useful feature
The list view for ingredient tracker was not modified, hence, there is no use to call the command i-list, hence this command is not useful

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: peter-yeh/ped#6

### LINKED ISSUE #182: [PE-D] Help Window too big
The help window is too big,  not sure if it happens on all screens. As shown in the screenshot
, this is my screen at full-sized. Some words below seems to have overflow out of my screen, the top menu bar to close and minimize window is overflowed too. I cannot move the window too, so my view is limited to this specific view as shown in the screenshot.

Will be nice if it is the size is draggable or the window is moveable.

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: kunnan97/ped#1

### LINKED ISSUE #183: [PE-D] Help Window to many words
Just to point out that normal users could be confused and shocked looking at the help window. This is not too big of an issue but would be nice if the window is kept clean. 

Suggestion could be to make the help window into different "pages" or with menubar at the top(like the one in AB3, "File" "Help) to filter the different category of commands. Similar was done in the UG command summary with different categories like "Sales Tracking" or "Ingredients Tracking"which was good. This will also help with the issue of "Help Window too big"  since there will be lesser content at once.

-------------
Labels: `severity.VeryLow` `type.FeatureFlaw`
original: kunnan97/ped#2

### LINKED ISSUE #202: [PE-D] Help window is too wordy
It is very daunting and difficult to look at so many words, perhaps a table and more concised wordings can be used so that it is easier for the user to read.

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: Cedo8/ped#9

### LINKED ISSUE #205: [PE-D] Command naming inconsistency
Both commands shown below are the provides the same functionality, but one is name as i-list and the other i-view-single. Why not name it i-list-single? I feel that i-view-single should be expanded to take in multiple ingredients as well, as viewing only 1 may not be as useful.

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: Cedo8/ped#12

### LINKED ISSUE #219: [PE-D] i-list does nothing
Same as the BBT list, as the GUI already shows the list, this i-list command does not do anything.

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: MatthiasLHK/ped#16

### LINKED ISSUE #220: [PE-D] i-show-ingrdient is not very useful
As the user can already see each ingredient from the GUI, the show command does not show anything new, so bascially it is kind of pointless.

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: MatthiasLHK/ped#17

### LINKED ISSUE #222: [PE-D] i-restock is not very useful
This is commandi is not very useful as it basically just show me the list of ingredients that are below the level. Which the user can alreadt see it from the GUI directly. Maybe instead of showing the amount currently, show how much more is needed.

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: MatthiasLHK/ped#19

### LINKED ISSUE #224: [PE-D] help command cannot be closed or exited after open
There is no closing prompt or option after entering the help command. 

-------------
Labels: `severity.High` `type.FeatureFlaw`
original: MatthiasLHK/ped#21

### PR DESCRIPTION
**Pull request by WM71811**

### CODE REVIEW

**Review by yanbingtao — DISMISSED** [review]
LGTM

File: src/main/java/seedu/address/model/ingredient/Ingredient.java [hunk 1]
```diff
@@ -62,6 +77,83 @@ public boolean isSameIngredient(Ingredient otherIngredient) {
...
+        if (this.isSolidIngredient()) {
+            return isSolidIngredientInShortage();
+        } else {
+            return isLiquidIngredientInShortage();
+        }
+    }
+
+    /**
+     * Returns true if the solid ingerdient is in shortage.
+     * @return true if the solid ingerdient is in shortage, false otherwise.
```
Comment (ureshiiYing) [inline src/main/java/seedu/address/model/ingredient/Ingredient.java]: There is a spelling error in "ingredient"

File: src/main/java/seedu/address/model/ingredient/Ingredient.java [hunk 2]
```diff
@@ -62,6 +77,83 @@ public boolean isSameIngredient(Ingredient otherIngredient) {
...
+        if (ingredientNames.contains(ingredientName)) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    /**
+     * Returns true if the ingerdient is in shortage, which is when their levels fall below the default level.
+     * @return true if the ingerdient is in shortage, false otherwise.
```
Comment (ureshiiYing) [inline src/main/java/seedu/address/model/ingredient/Ingredient.java]: There is a spelling error in "ingredient"

File: src/main/java/seedu/address/model/ingredient/Ingredient.java [hunk 3]
```diff
@@ -62,6 +77,83 @@ public boolean isSameIngredient(Ingredient otherIngredient) {
...
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    /**
+     * Returns true if the liquid ingerdient is in shortage.
+     * @return true if the liquid ingerdient is in shortage, false otherwise.
+     */
```
Comment (ureshiiYing) [inline src/main/java/seedu/address/model/ingredient/Ingredient.java]: There is a spelling error in "ingredient"

File: src/main/java/seedu/address/model/ingredient/IngredientName.java [hunk 4]
```diff
@@ -38,6 +38,14 @@ public static boolean isValidIngredientName(String ingredientName) {
...
+     * Returns the name of the ingredient in string format.
+     * @return the name of the ingredient in string format.
+     */
+    public String getIngredientNameString() {
+        return ingredientName;
+    }
+
     @Override
     public String toString() {
         return ingredientName;
```
Comment (ureshiiYing) [inline src/main/java/seedu/address/model/ingredient/IngredientName.java]: Difference between these two functions?

**Review by ureshiiYing — COMMENTED** [review]
The rest are LGTM

File: src/main/java/seedu/address/model/ingredient/Ingredient.java (hunk 1 above)
Comment (WM71811) [reply src/main/java/seedu/address/model/ingredient/Ingredient.java]: Noted, thanks!

File: src/main/java/seedu/address/model/ingredient/Ingredient.java (hunk 2 above)
Comment (WM71811) [reply src/main/java/seedu/address/model/ingredient/Ingredient.java]: Noted, thanks!

File: src/main/java/seedu/address/model/ingredient/Ingredient.java (hunk 3 above)
Comment (WM71811) [reply src/main/java/seedu/address/model/ingredient/Ingredient.java]: Noted, thanks!

File: src/main/java/seedu/address/model/ingredient/IngredientName.java (hunk 4 above)
Comment (WM71811) [reply src/main/java/seedu/address/model/ingredient/IngredientName.java]: They are the same, thanks for noting. I will delete the function above.

**Review by yanbingtao — APPROVED** [review]
LGTM

**Review by ureshiiYing — APPROVED** [review]
LGTM
